### ISSUE TITLE
Shortcut not created after Restore

### ISSUE THREAD

**Issue by edmondt** [Simizfo/WSATools#47]
Backup and Restore is a great feature! Thank you :) 

The Apps are not on the shortcut after restore tho... Any workaround?

**Comment by edmondt** [comment]
Edit: found installing this and clicking on each app will create the shortcut on the start menu: https://github.com/infinitepower18/WSA-InstalledApps/releases/tag/v2.0.0

**Comment by midi1996** [comment]
Experiencing the same problem, backed up my wsa setup, did a fresh install of windows, restoring the backup does not refresh the app list in start, you do that you have to manually reinstall every single app.

**Comment by arkni8** [comment]
Yep, seems to be the case, whether you used this app or manually backed up the userdata file.
But, if you go to Windows Installed App, you will get the list of installed android app. You can access their settings from there and open the apps, I guess.

This WSA-Installed-Apps is also pretty good, but again, if you backed it up and restore, even its icon will disappear from start menu and then you will need to reinstall that application atleast

**Comment by Simizfo** [comment]
When I developed the backup feature, yes, the apps didn't immediately appear on the start menu, but they did after a reboot. I'll give it a try soon to see if this has changed. Thanks for reporting!
